packages/components/checkbox/src/checkbox.ts

```diff
@@ -182,7 +190,7 @@ const useCheckboxStatus = (
     if (toTypeString(value) === '[object Boolean]') {
       return value
     } else if (Array.isArray(value)) {
-      return value.includes(props.label)
+      return value.map((o) => toRaw(o)).includes(props.label)
```

Comment:
It seems that it can be abbreviated 😆
```suggestion
      return value.map(toRaw).includes(props.label)
```

Reply:
You are right, I learned it again.